src/main/java/frc/robot/subsystems/intake/IntakeCMD.java

```diff
@@ -1,46 +1,76 @@
 package frc.robot.subsystems.intake;
 
-import java.util.function.DoubleSupplier;
-
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.led.LedSS;
 import frc.robot.constants.RobotConstants.IntakeConstants;
+import frc.robot.constants.RobotConstants.SpinnerConstants;
+import frc.robot.utilities.DriverStationLogger;
+
+import java.util.function.DoubleSupplier;
 
 public class IntakeCMD extends CommandBase {
     private final IntakeSS intakeSS;
     private final IntakeConstants constants;
-    private final LedSS ledSS;
-    private final DoubleSupplier power;
+    private DoubleSupplier power;
+    private double output;
+    private double initialOutput;
+    private double reverseMotorStartTime;
+    private boolean stillStalled;
+    private boolean stallLogic;
 
-    public IntakeCMD(IntakeSS intakeSS, LedSS ledSS, IntakeConstants constants, DoubleSupplier power) {
+    public IntakeCMD(IntakeSS intakeSS, IntakeConstants constants, DoubleSupplier power, boolean stallLogic) {
         this.intakeSS = intakeSS;
         this.constants = constants;
-        this.ledSS = ledSS;
         this.power = power;
+        this.stallLogic = stallLogic;
         addRequirements(intakeSS);
     }
 
-    public IntakeCMD(IntakeSS intakeSS, LedSS ledSS, IntakeConstants constants, double power) {
-        this(intakeSS, ledSS, constants, () -> power);
+    public IntakeCMD(IntakeSS intakeSS, IntakeConstants constants, DoubleSupplier power) {
+        this(intakeSS, constants, power, true);
     }
 
-    public IntakeCMD(IntakeSS intakeSS, LedSS ledSS, IntakeConstants constants) {
-        this(intakeSS, ledSS, constants, () -> constants.DEFAULT_MOTOR_POWER);
+    public IntakeCMD(IntakeSS intakeSS, IntakeConstants constants) {
+        this(intakeSS, constants, () -> constants.DEFAULT_MOTOR_POWER);
```

Comment:
How is this not giving you an error? you aren't giving the supplier a `usingStallLogic`